### PR TITLE
Update memory-default-namespace.md

### PR DESCRIPTION
**Pull request by avgKol**
https://github.com/kubernetes/kubernetes/issues/51430

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
> Help editing and submitting pull requests:  https://deploy-preview-9510--kubernetes-io-master-staging.netlify.com/docs/contribute/start/#submit-a-pull-request.
> Help choosing which branch to use, see
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
LGTM. Defer to @derekwaynecarr for further tech accuracy ;)

**Comment by zacharysarah** [comment]
@vishh 👋 PTAL

File: content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 1]
```diff
@@ -149,6 +149,9 @@ resources:
   requests:
     memory: 128Mi
 ```
+{{< note >}}
+**Note**: Kubernetes rejects the  Container's memory request if it exceeds the default memory request specified by the LimitRange.
```
Comment (derekwaynecarr) [inline content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: i do not think this is accurate.

if a container makes no memory request, but a limit range has a default request, the container picks up the default request.

if a container has a memory request, the limit range should only evaluate that request relative to the min constraint.

Comment (vishh) [reply content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: +1 to what @derekwaynecarr mentioned. This is inaccurate.

**Comment by zacharysarah** [comment]
@vishh @derekwaynecarr Thanks for the feedback! 🙇 

@avgKol Please feel free to reopen this PR when you're ready to address review feedback. ☝️

**Pull request closed without merging**